src/models/continuity/ContETFStatic.ts

```diff
@@ -50,7 +54,36 @@ export default class ContETFStatic
     startDot: StuntSheetDot,
     endDot?: StuntSheetDot
   ): void {
-    // TODO: Implement
+    let dx = 0;
+    let dy = 0;
+    switch (this.marchingDirection) {
+      case DIRECTIONS.E: {
+        dy = 1;
+        break;
+      }
+      case DIRECTIONS.W: {
+        dy = -1;
+        break;
+      }
+      case DIRECTIONS.N: {
+        dx = 1;
+        break;
+      }
+      case DIRECTIONS.S: {
+        dx = -1;
+        break;
+      }
+    }
+    const [x, y]: [number, number] = startPositionHelper(flow, startDot);
+    for (let beat = 1; beat <= Math.max(this.duration, 1); beat += 1) {
```

Comment:
Looking back at `Math.max(this.duration, 1)`... Maybe it should be `Math.max(this.duration, 0)`... If the duration is 0 then the marcher should not move at all.

Reply:
This makes sense. I was going off ContInPlace (which has a minimum of 1), but having a minimum of zero for etfstatic makes sense 👍